utils/config-utils.js

```diff
@@ -1,5 +1,26 @@
 import { version } from '_/package.json';
 
+async function initDevtools() {
+  try {
+    // Check devtools availability
+    const response = await fetch('/api/devtools-health');
+    const { alive } = await response.json();
+    if (!alive) {
+      throw new Error('Devtools not available');
+    }
+    // Add devtools script to the DOM
+    const script = document.createElement('script');
+    script.src = 'http://localhost:8098';
+    script.async = true;
+    script.onerror = () => {
+      console.error('Failed to load the devtools script');
+    };
+    document.head.appendChild(script);
+  } catch (err) {
+    console.error('Error checking devtools availability:', err.message);
```

Comment:
I think this should probably be a warning instead of an error
```suggestion
    console.warn('Error checking devtools availability:', err.message);
```